src/vs/workbench/contrib/terminalContrib/suggest/test/browser/terminalCompletionService.test.ts

```diff
@@ -113,15 +161,40 @@ suite('TerminalCompletionService', () => {
 				pathSeparator
 			};
 			const result = await terminalCompletionService.resolveResources(resourceRequestConfig, './', 3, provider);
-			assert.deepEqual(result, [{
-				label: `.${pathSeparator}folder1${pathSeparator}`,
-				provider,
-				kind: TerminalCompletionItemKind.Folder,
-				isDirectory: true,
-				isFile: false,
-				replacementIndex: 1,
-				replacementLength: 2
-			}]);
+
+			interface IAssertionTerminalCompletion {
+				label: string;
+				detail?: string;
+				kind?: TerminalCompletionItemKind;
+			}
+
+			interface IAssertionCommandLineConfig {
+				replacementIndex: number;
+				replacementLength: number;
+			}
+
+			function assertCompletions(actual: ITerminalCompletion[] | undefined, expected: IAssertionTerminalCompletion[], expectedConfig: IAssertionCommandLineConfig) {
+				assert.deepStrictEqual(
+					actual?.map(e => ({
+						label: e.label,
```

Comment:
Let's replace `/` with `pathSeparator` here? That'll make tests look even nicer.